packages/vite-plugin-cloudflare/package.json

```diff
@@ -45,6 +45,7 @@
 		"@cloudflare/unenv-preset": "workspace:*",
 		"@hattip/adapter-node": "^0.0.49",
 		"miniflare": "workspace:*",
+		"picocolors": "^1.1.1",
```

Comment:
I think this one should be dev dependency so that it's bundled. Following Vite here.